QA/conf/fetch_api_interviewscheduler_conf.py

```diff
@@ -0,0 +1,6 @@
+"""
+This page provides configuration details to provide in order to fetch the api response
+"""
+emailid="test@qxf2.com"
+date="07/10/2025"
+url="http://127.0.0.1:6464/get-schedule"
```

Comment:
We dont add URL in the conf file.is there any specific reason you have included URL here?

Reply:
I have deleted the url from here.